src/Console/Command/HelpCommand.php

```diff
@@ -51,15 +51,15 @@ public static function toString($value): string
     /**
      * Returns the allowed values of the given option that can be converted to a string.
      *
-     * @return list<null|AllowedValueSubset|scalar>
+     * @return null|list<AllowedValueSubset|mixed>
```

Comment:
```suggestion
     * @return null|list<mixed>
```

Reply:
i was thinking about it too. yet `AllowedValueSubset` is unusual, precise, custom-made type that we often return here, and for that maybe it's worth to explicitly mention it.

but no strong feelings.